src/components/toast/toast.style.ts

```diff
@@ -129,24 +158,24 @@ const ToastStyle = styled(MessageStyle)<{
       }
 
       &${alternativeAnimationName}-appear, &${alternativeAnimationName}-enter {
-        bottom: -40px;
+        ${alignY === "top" ? "top: -40px;" : "bottom: -40px;"};
```

Comment:
```suggestion
        ${alignY === "top" ? "top: -40px" : "bottom: -40px"};
```
Comment (non-blocking): this id adding an additional `;` here, I believe v4 of styled-components handles these but v5 doesn't so we'd have issues if/ when we upgraded